README.md

```diff
@@ -38,21 +38,22 @@ Lovelace Button card for your entities.
+| `show_name`    | boolean     | `true`       | `true` \| `false`                                | Wether to show the name or not. Will pick entity_id's name by default, unless redefined in the `name` property or in any state `name` property                                                                                                                                                                                                |
+| `show_state`   | boolean     | `false`      | `true` \| `false`                                | Show the state on the card. defaults to false if not set                                                                                                                                                                                                                                                                                      |
+| `show_icon`    | boolean     | `true`       | `true` \| `false`                                | Wether to show the icon or not. Unless redefined in `icon`, uses the default entity icon from hass                                                                                                                                                                                                                                            |
+| `style`        | object list | optional     | `- text-transform: none`                         | Define a list of css attribute and their value to apply to the card                                                                                                                                                                                                                                                                           |
+| `state`        | object list | optional     | See [State](#State)                              | State to use for the color, icon and style of the button. Multiple states can be defined                                                                                                                                                                                                                                                      |
+| `confirmation` | boolean     | `false`      | `true` \| `false`                                | Show a confirmation popup on tap                                                                                                                                                                                                                                                                                                              |
```

Comment:
You don't need to worry about this spacing unless it makes it easier for you to edit. I just bring up the preview pane in VS Code.

Reply:
Yeah, the plug-in I use does that automatically but it's a mess to merge on conflicts and not very clear about what really changed, I'm going to disable this feature :)